### DISCUSSION TITLE
Star Sports 1

### DISCUSSION THREAD

**Discussion by DPX911** [iptv-org/community#924] [Q&A]
### Channel ID

StarSports1Tamil.in

### Notes (optional)

Channel name: Star Sports 1 Tamil

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19858
